src/simsopt/geo/curve.py

```diff
@@ -813,7 +864,8 @@ def flip(self):
         return True if self.rotmat[2][2] == -1 else False
 
 
-def curves_to_vtk(curves, filename, close=False, extra_data=None):
+def curves_to_vtk(curves, filename, close=False, I=None, extra_point_data=None,
+                  NetForces=None, NetTorques=None):
```

Comment:
I wouldn't rename extra_data - that will break other people's scripts. 

It doesn't smell right to add current and forces and torques here. These are field concepts, not geo concepts.

Instead of adding `I` here, how about making a `to_vtk` function for Coils, which delegates to Curve.to_vtk after adding the current data to extra_data. Maybe force and torque could be added there. (And use snake_case)